runtime/runtime-params-estimator/src/estimator_context.rs

```diff
@@ -31,6 +40,20 @@ pub(crate) struct CachedCosts {
     pub(crate) apply_block: Option<GasCost>,
 }
 
+/// A raw runtime to estimate costs that don't need anything external. Also
+/// allows to compare VMs using the --vm-kind option.
+pub(crate) struct EstimatorRuntime {
```

Comment:
Let's call it `WASM` runtime (or, well, contract runtime), that's more descriptive than Raw or estimator